newsrevenuehub/rev-engine
# DEV-2724: Add integration to AvatarMenu

Please fill out each section even if it's just with "N/A"

#### Plan? (if this draft/incomplete indicate what you intend to do and how)
n/a
#### What's this PR do?
- Adds "Integrations" to AvatarMenu
- Redirects "settings/integrations" back to "pages" if user has more than 1 org
- Update pre-commit yaml isort version to fix committing issues

#### Why are we doing this? How does it help us?
This makes the "settings/integration" page accessible for the user within the webapp and adds some business logic around that page.

#### How should this be manually tested? Please include detailed step-by-step instructions.
Test all 3 scenarios:
1) - Log in with HUB admin
    - Check that AvatarMenu does not have Integrations button
    - Access "settings/integrations" in the URL and make sure it works

2) - Log in with user with access to multiple Orgs
    - Check that AvatarMenu does not have Integrations button
    - Access "settings/integrations" in the URL and make sure it redirects to "/pages"

3) - Log in with user with access to 1 Org
    - Check that AvatarMenu has Integrations button and that it works (redirects to "settings/integrations")
    - Access "settings/integrations" in the URL and make sure it works

#### Did this PR make changes to the user interface that may require changes to the user-facing docs?
yes

#### Have automated unit tests been added? If not, why?
yes

#### How should this change be communicated to end users?
Integrations now added to Avatar Menu

#### Are there any smells or added technical debt to note?
no

#### Has this been documented? If so, where?
no

#### What are the relevant tickets? Add a link to any relevant ones.
https://news-revenue-hub.atlassian.net/browse/DEV-2724

#### Do any changes need to be made before deployment to production (adding environment variables, for example)?
no

#### Are there next steps? If so, what? Have tickets been opened for them? List next-step tickets here:
no
